### PR TITLE
Add me.broken_by.PdfMetadataEditor

### PR DESCRIPTION
**Pull request by zaro**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: me.brokenby.PdfMetadataEditor.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+app-id: me.brokenby.PdfMetadataEditor
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:1]: brokenby.me doesn't seem to resolve. It has to be domain under your control.

File: flatpak/me.brokenby.PdfMetadataEditor-mime.xml
Comment (bbhtt) [inline flatpak/me.brokenby.PdfMetadataEditor-mime.xml:1]: This should be upsteam

File: flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop
Comment (bbhtt) [inline flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop:1]: This should be upsteam

File: flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Batch Pdf Metadata Editor
+Comment=Simple Editor for PDF metadata (Batch Mode)
+Exec="/app/bin/Batch Pdf Metadata Editor"
```
Comment (bbhtt) [inline flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop:4]: Don't use absolute path and preferably name the executable something that doesn't have spaces

File: flatpak/me.brokenby.PdfMetadataEditor.desktop
Comment (bbhtt) [inline flatpak/me.brokenby.PdfMetadataEditor.desktop:1]: Same comments as other desktop file. Why are there two of these?

File: flatpak/me.brokenby.PdfMetadataEditor.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.1-->
+  <id>me.brokenby.PdfMetadataEditor</id>
+  <name>Pdf Metadata Editor</name>
+  <summary>Edit PDF metadata</summary>
+  <developer_name>Svetlozar Argirov</developer_name>
+  <metadata_license>MIT</metadata_license>
```
Comment (bbhtt) [inline flatpak/me.brokenby.PdfMetadataEditor.metainfo.xml:8]: Add project_license

File: flatpak/pdf-metadata-edit.icns
Comment (bbhtt) [inline flatpak/pdf-metadata-edit.icns:1]: Why are these here?

File: flatpak/pdf-metadata-edit.ico
Comment (bbhtt) [inline flatpak/pdf-metadata-edit.ico:1]: Why are these here?

File: flatpak/pdf-metadata-edit.png
Comment (bbhtt) [inline flatpak/pdf-metadata-edit.png:1]: This should be upstream

File: me.brokenby.PdfMetadataEditor.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - '/app/bin/pmedit-cli'
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:6]: The command doesn't match the desktop file Exec?

File: me.brokenby.PdfMetadataEditor.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - mkdir -p /app/share/mime/packages/
+      - mkdir -p /app/share/applications/
+      - cp -v pdf-metadata-edit.png /app/share/icons/hicolor/128x128/apps/me.brokenby.PdfMetadataEditor.png
+      - cp -v me.brokenby.PdfMetadataEditor-mime.xml /app/share/mime/packages/me.brokenby.PdfMetadataEditor.xml
+      - cp -v *.desktop /app/share/applications/
+    sources:
+      - type: dir
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:26]: Don't use dir

File: me.brokenby.PdfMetadataEditor.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+finish-args:
+  [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
+    "--persist=.java",
+  ]
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:38]:
```suggestion
finish-args:
  - --share=ipc
  - --socket=x11
  - --socket=pulseaudio
  - --share=network
  - --device=dri
  - --filesystem=home
  - --persist=.java
```

Comment (bbhtt) [reply me.brokenby.PdfMetadataEditor.yml:38]: This should be top level after commands before modules.

File: me.brokenby.PdfMetadataEditor.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        path: flatpak
+
+finish-args:
+  [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:36]: Why do you need full home? Does it support filechooster portals?

File: me.brokenby.PdfMetadataEditor.yml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+      - cp -v *.desktop /app/share/applications/
+    sources:
+      - type: dir
+        path: flatpak
+
+finish-args:
+  [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:33]: Why pulseaudio? It is a pdf editor?

File: me.brokenby.PdfMetadataEditor.yml [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    "--share=network",
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:34]: Why network?

File: me.brokenby.PdfMetadataEditor.yml [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+commands:
+  - '/app/bin/pmedit-cli'
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
```
Comment (bbhtt) [inline me.brokenby.PdfMetadataEditor.yml:14]: Is this x86_64 only? Why not build from source? Is aarch64 available?

If you want x86_64 only you need a flathub.json https://docs.flathub.org/docs/for-app-authors/maintanance#only-build-on-x86_64

File: flatpak/pdf-metadata-edit.icns
Comment (hfiguiere) [reply flatpak/pdf-metadata-edit.icns:1]: This is for macOS, remove.

File: flatpak/pdf-metadata-edit.ico
Comment (hfiguiere) [reply flatpak/pdf-metadata-edit.ico:1]: this is for Windows. Remove.

File: me.brokenby.PdfMetadataEditor.yml (hunk 1 above)
Comment (hfiguiere) [reply me.brokenby.PdfMetadataEditor.yml:1]: I think it should be, based on the appstream file, `me.broken_by.*`

File: me.brokenby.PdfMetadataEditor.yml (hunk 4 above)
Comment (hfiguiere) [reply me.brokenby.PdfMetadataEditor.yml:6]: command is of type "string" not "array". Flatpak-builder will, at best, complain if not simply bail out. Did you build this locally?

File: me.brokenby.PdfMetadataEditor.yml [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+      - mkdir -p /app/share/mime/packages/
+      - mkdir -p /app/share/applications/
+      - cp -v pdf-metadata-edit.png /app/share/icons/hicolor/128x128/apps/me.brokenby.PdfMetadataEditor.png
+      - cp -v me.brokenby.PdfMetadataEditor-mime.xml /app/share/mime/packages/me.brokenby.PdfMetadataEditor.xml
+      - cp -v *.desktop /app/share/applications/
+    sources:
+      - type: dir
+        path: flatpak
+
+finish-args:
```
Comment (hfiguiere) [inline me.brokenby.PdfMetadataEditor.yml:29]: `finish-args` are usually found above `modules`

File: me.brokenby.PdfMetadataEditor.yml [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+  [
```
Comment (hfiguiere) [inline me.brokenby.PdfMetadataEditor.yml:30]: please use proper YAML syntax. Or since you seem to prefer JSON syntax why not making the manifest JSON?

File: me.brokenby.PdfMetadataEditor.yml (hunk 1 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: The domain is broken-by,me , and since it didn't accept the '-' i simply removed it .  Fixed in latest commit.

File: flatpak/me.brokenby.PdfMetadataEditor-mime.xml
Comment (zaro) [reply flatpak/me.brokenby.PdfMetadataEditor-mime.xml:1]: moved

File: flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop
Comment (zaro) [reply flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop:1]: moved

File: flatpak/me.brokenby.PdfMetadataEditor.desktop
Comment (zaro) [reply flatpak/me.brokenby.PdfMetadataEditor.desktop:1]: Because  there are two main interfaces , one is single file the other one edits many files in batch mode.

Anyway, removed this .desktop file for now as there are other ways to launch it from the other one.

File: me.brokenby.PdfMetadataEditor.yml (hunk 8 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: Removed, just blind copy paste.

File: me.brokenby.PdfMetadataEditor.yml (hunk 9 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: Copy/paste. removed.

File: me.brokenby.PdfMetadataEditor.yml (hunk 11 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: moved.

File: me.brokenby.PdfMetadataEditor.yml (hunk 12 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: Changed, to YAML. Just copy paste artefact, I definitely prefer YAML :)

File: flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop (hunk 2 above)
Comment (zaro) [reply flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop]: Removed the absolute path.

Unfortunately due to a limitation of jpackage ( the tool used to build the program) I can't make the executable name different than the general name,  and this results in executable with spaces.

Once jpackage is fixed, I for sure will change the executable name,

File: me.brokenby.PdfMetadataEditor.yml (hunk 6 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: moved.

File: me.brokenby.PdfMetadataEditor.yml (hunk 7 above)
Comment (zaro) [reply me.brokenby.PdfMetadataEditor.yml]: I am really not sure how to answer . If I remove the --filesystem=home, the home directory  is empty, and the whole purpose of the tool is to edit existing files. 

Not sure also about the filechooser portals, but unless Java 17 supports them the answer is probably no.

File: me.brokenby.PdfMetadataEditor.yml (hunk 1 above)
Comment (hfiguiere) [reply me.brokenby.PdfMetadataEditor.yml]: The doc says to replace `-` by `_`.

https://docs.flatpak.org/en/latest/conventions.html#application-ids

File: flathub.json [hunk 13]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing. Since this is written in JAVA it should just run on an architecture, isn't it? "Write once, run everywhere" has been the slogan for Java for over 25 years now.

File: flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml [hunk 14]
```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with
+  jdAppStreamEdit 7.1-->
+  <id>me.broken_by.PdfMetadataEditor</id>
+  <name>Pdf Metadata Editor</name>
+  <summary>Edit PDF metadata</summary>
+  <developer_name>Svetlozar Argirov</developer_name>
+  <metadata_license>GFDL-1.3</metadata_license>
```
Comment (hfiguiere) [inline flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml:9]: CC0-1.0 is prefered for the metadata.

This file should also be managed upstream for future releases.

File: me.broken_by.PdfMetadataEditor.yml [hunk 15]
```diff
@@ -0,0 +1,38 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - "/app/bin/pmedit-cli"
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:6]:
```suggestion
  - "pmedit-cli"
```

Not an absolute path.

File: me.broken_by.PdfMetadataEditor.yml [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --persist=.java
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:12]: with the broad `home` permission you don't need persist.

File: me.broken_by.PdfMetadataEditor.yml [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/archive/refs/heads/master.zip
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:35]: you are referencing `master` which mean if there is a commit between now and then, this will break.

File: me.broken_by.PdfMetadataEditor.yml [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:32]:
```suggestion
      - install -Dm644 flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
      - install -Dm644 flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
      - install -Dm644 flatpak/me.broken_by.PdfMetadataEditor.desktop -t ${FLATPAK_DEST}/share/applications/
      - install -Dm644 me.broken_by.PdfMetadataEditor.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
```

File: me.broken_by.PdfMetadataEditor.yml [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
+        sha256: 80303e459c1c3b6ecf13923545ca349a2c3e1ff2a3ede08644adba98b2a9de2d
+  - name: pdf-metadata-editor-desktop
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:22]: this should only be one single module

File: me.broken_by.PdfMetadataEditor.yml (hunk 18 above)
Comment (hfiguiere) [reply me.broken_by.PdfMetadataEditor.yml:32]: also I shall add that none of these files are flatpak specific so wherever you have put them in the repoistory is confusing. Not that it matters here, but I have to debunk that myth.

File: flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml (hunk 14 above)
Comment (hfiguiere) [reply flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml:9]: just to had, this file is NOT flatpak specific either.

File: me.broken_by.PdfMetadataEditor.yml (hunk 18 above)
Comment (hfiguiere) [reply me.broken_by.PdfMetadataEditor.yml:32]: also the .desktop file, you didn't fix the `Exec` line. (you ignored the review comment)

File: me.brokenby.PdfMetadataEditor.yml (hunk 10 above)
Comment (bbhtt) [reply me.brokenby.PdfMetadataEditor.yml]: You have to manually add all maven sources by hand, there's a few apps doing it I think but maven dependencies can be a bit of hit or miss because some of them assume a network connection is present at build time which isn't true on Flathub.

File: flathub.json (hunk 13 above)
Comment (zaro) [reply flathub.json:2]: There was no need/request so far by anybody for an ARM version, hence it's missing.

And clearly building Java applications inside flatpak is a PITA because maven can't download dependencies.
Once this is merged https://github.com/flatpak/flatpak-builder-tools/pull/253 , I'll probably change it to be built from source and this problem will go away.

File: me.broken_by.PdfMetadataEditor.yml (hunk 17 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml:35]: yes, I understand this is a problem. My intention was once all the other feedback is resolved to tag a new release and use it in place of master.

File: me.broken_by.PdfMetadataEditor.yml (hunk 16 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: removed

File: me.broken_by.PdfMetadataEditor.yml (hunk 15 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: changed accordinly

File: flathub.json (hunk 13 above)
Comment (hfiguiere) [reply flathub.json:2]: This still doesn't answer why it's exclusive to x86_64 when Java is supposed to be write once run everywhere.

File: me.broken_by.PdfMetadataEditor.yml (hunk 17 above)
Comment (hfiguiere) [reply me.broken_by.PdfMetadataEditor.yml:31]: its a chicken and egg problem because this needs to be resolved.

File: me.broken_by.PdfMetadataEditor.yml [hunk 20]
```diff
@@ -0,0 +1,35 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - pmedit-cli
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:6]: there is no property `commands`. I'm pretty sure flatpak-builder threw a warning when you tried to build this.

File: me.broken_by.PdfMetadataEditor.yml [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+    build-commands:
+      - mv -v app/* ${FLATPAK_DEST}
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v src/flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v src/flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v src/flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:24]: I suggested changes here that you discarded.

File: flathub.json (hunk 13 above)
Comment (zaro) [reply flathub.json:2]: I don't want to get into arguments about Java honestly, I was simply trying to explain why currently there is only x86 version and not an ARM version.

But to add a bit more information,  yes the JAR runs everywhere. BUT, the desktop integration part that includes a JRE , creates convenient binaries and so on is platform specific and need to be rebuild for each platform.

Also i want to point out, this is  a side project for me I only spend time maintaining it because people find it useful. My daily work doesn't involve Java at all, so my knowledge of many things about Java are mediocre at best.

File: flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml (hunk 14 above)
Comment (zaro) [reply flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml]: Changed

File: me.broken_by.PdfMetadataEditor.yml (hunk 17 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml:31]: As I said I will definitely resolve once it's the last item in the PR left.

I hope that is fine by you ?

File: me.broken_by.PdfMetadataEditor.yml (hunk 18 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: I for sure haven't heard about that myth , so no chance of it sticking on me now :)

Regarding the .desktop file , what do you mean ?
I did change it and removed the absolute path. The other comment was about the spaces in the binary name, but as I tried to explain in the other comment this is a limitation of the `jpackage` tool I am using to build the software.  I don't like it at all that there are spaces, but I am not sure what to do about it, because the tool doesn't have separate options for Application name and Binary name, you have to pick the same for both.

Is this a huge problem? it seems to work just fine.

File: me.broken_by.PdfMetadataEditor.yml (hunk 19 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: done

File: me.broken_by.PdfMetadataEditor.yml (hunk 21 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: Yes, sorry about that. Added them.

File: me.broken_by.PdfMetadataEditor.yml (hunk 20 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: yes, you are right. 
Thanks, fixed.

File: me.broken_by.PdfMetadataEditor.yml [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+  - --filesystem=home
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v app/* ${FLATPAK_DEST}
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
```
Comment (hfiguiere) [inline me.broken_by.PdfMetadataEditor.yml:19]:
```suggestion
```

My suggestion was also to remove these.

File: me.broken_by.PdfMetadataEditor.yml (hunk 17 above)
Comment (hfiguiere) [reply me.broken_by.PdfMetadataEditor.yml:30]: no. pin to a commit.

File: me.broken_by.PdfMetadataEditor.yml (hunk 22 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: I see. I am sorry I kind of glazed over that.

File: me.broken_by.PdfMetadataEditor.yml (hunk 17 above)
Comment (zaro) [reply me.broken_by.PdfMetadataEditor.yml]: Pinned to new release.

**Comment by bbhtt** [comment]
bot, build me.brokenby.PdfMedataEditor

**Comment by bbhtt** [comment]
You should change the title to match the app id

bot, build me.broken_by.PdfMetadataEditor
